test/e2e/simple/config.yaml

```diff
@@ -18,6 +18,10 @@ tests:
   commands: exit 1
   container:
     from: os
+- as: cloning
```

Comment:
Maybe call this implicit cloning since it's from a base image, and add another test for explicit cloning from some other image? Can we test for the content of the repo?

Reply:
Updated to have an implicit cloning test and one where we disable it.